lib/cocina/generator/schema_base.rb

```diff
@@ -60,31 +61,54 @@ def relaxed_comment
       end
 
       def dry_datatype(doc)
+        return doc.name if doc.name.present? && schemas.include?(doc.name)
+
+        datatype_from_doc_type(doc) ||
+          datatype_from_doc_names(doc) ||
+          raise("#{doc.type} not supported")
+      end
+
+      def datatype_from_doc_type(doc)
         case doc.type
         when 'integer'
-          'Strict::Integer'
+          'Types::Strict::Integer'
         when 'string'
           string_dry_datatype(doc)
         when 'boolean'
-          'Strict::Bool'
+          'Types::Strict::Bool'
         else
-          return 'Nominal::Any' if any_datatype?(doc)
+          return 'Types::Nominal::Any' if any_datatype?(doc)
```

Comment:
This is a bug I introduced in the prior PR earlier today while trying to break one PR into several. Fixing that here.

Reply:
bummer about the bug.  Thanks for the fix.